redux/actions/stories.actions.ts

```diff
@@ -9,19 +9,7 @@ export const NEW_STORY = 'NEW_STORY';
 export const SAVED_NEW_STORY = 'SAVED_NEW_STORY';
 export const CLEAR_NEW_STORY = 'CLEAR_NEW_STORY';
 
-interface AddStories {
-  id: string;
-  stories: Record<string, Story[]>;
-}
-
-interface EditStory {
-  projectId: string;
-  storyState: string;
-  story: Story;
-}
-
 interface MoveStories {
```

Comment:
@jansen0825 I mostly moved away from the payload pattern here. Happy to add it back if you want, but I think this simplifies it, I also was able to get most of the arguments down to 1 argument.